src/tests/system/tests/test_authentication.py

```diff
@@ -137,3 +137,58 @@ def test_authentication__using_the_users_email_address(client: Client, ad: AD, m
     assert client.auth.parametrize(method).password(
         "uSEr_3@alias-dOMain.com", "Secret123"
     ), "User uSEr_3@alias-dOMain.com failed login!"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.ticket(jira="RHEL-65848", gh=7690)
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.importance("high")
+def test_authentication__by_updating_id_provider_to_ldap(client: Client, provider: GenericADProvider, method: str):
```

Comment:
I think this should go into 'test_ad.py' file and have a name that is a little more general like 'test_ad__user_authentication_when_provider_is_set_to_ldap_with_gss_spnego' . 

Is this test about if it's configured with a different provider, or does it have to be set to ad_provider first? 

FYI, similar tests cover using the ldap provider to connect to AD with other auth providers, having the provider and auth mech in the name will be helpful.

Reply:
Yes, I can move it to test_ad.py, but I added in test_authentication.py because we are checking auth in this case.

It's with ad and samba provider where default `id_provider = ad`

Reply:
I updated, please check.